### PR TITLE
[Pages] emberjs framework guide

### PR DESCRIPTION
**Pull request by deadlypants1973**
closes WDC-899

### CODE REVIEW

File: content/pages/framework-guides/deploy-an-emberjs-site.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+```sh
+$ ember new ember-quickstart --lang en
+```
+
+{{<render file="_tutorials-before-you-start.md">}}
+
+## Create a GitHub repository
+
+Create a new GitHub repository by visiting [repo.new](https://repo.new). When creating your repository, do not select to add a README file, .gitignore template or a license as these selections will cause the push to GitHub to fail.
```
Comment (marciocloudflare) [inline content/pages/framework-guides/deploy-an-emberjs-site.md]:
```suggestion
Create a new GitHub repository by visiting [repo.new](https://repo.new). When creating your repository, do not select to add a README file, `.gitignore` template or a license as these selections will cause the push to GitHub to fail.
```

File: content/pages/framework-guides/deploy-an-emberjs-site.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+You have successfully created a GitHub repository and pushed your Ember project to that repository.
+
+## Deploy with Cloudflare Pages
+
+Deploy your site to Pages:
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
+2. In **Account Home**, select **Pages** > **Create a project**. 
+3. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, select *Ember* as your **Framework preset**. Your selection will provide the following information.
```
Comment (marciocloudflare) [inline content/pages/framework-guides/deploy-an-emberjs-site.md]:
```suggestion
3. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, select *Ember* as your **Framework preset**. Your selection will provide the following information:
```

File: content/pages/framework-guides/deploy-an-emberjs-site.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+| -------------------- | --------------------------- |
+| Production branch    | `main`                      |
+| Build command        | `ember build`               |
+| Build directory      | `dist`                      |
+
+{{</table-wrap>}}
+
+After deploying your site, you will receive a unique subdomain for your project on `*.pages.dev`.
+
+Every time you commit new code to your Ember site, Cloudflare Pages will automatically rebuild your project and deploy it. You will also get access to [preview deployments](/pages/platform/preview-deployments/) on new pull requests and be able to preview how changes look to your site before deploying them to production.
```
Comment (marciocloudflare) [inline content/pages/framework-guides/deploy-an-emberjs-site.md]: Not sure if it's the non-native in me, but `and be able to preview how changes look to your site before` sounds weird. To me it sounds better to say `and be able to preview how changes to your site look before`. Wdyt?

**Review by marciocloudflare — APPROVED** [review]

File: content/pages/framework-guides/deploy-an-emberjs-site.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-an-emberjs-site.md]:
```suggestion
Every time you commit new code to your Ember site, Cloudflare Pages will automatically rebuild your project and deploy it. You will also get access to [preview deployments](/pages/platform/preview-deployments/) on new pull requests and be able to preview how changes to your site look before deploying them to production.
```

**Pull request merged**